code/game/objects/items/clown_items.dm

```diff
@@ -246,3 +246,47 @@
 	desc = "Just looking at this makes you want to giggle."
 	icon_state = "laughter"
 	list_reagents = list(/datum/reagent/consumable/laughter = 50)
+
+/obj/item/rubber_chicken
+	name = "rubber chicken"
+	desc = " A robust rubber chicken the size of a Louisiana broiler.\nIt looks somewhat clammy and exudes a noticable rubbery smell.\nA small peeling sticker on the chicken's left thigh reads: 'made by waffle co.'"
+	icon = 'icons/obj/clown_items.dmi'
+	icon_state = "rubber_chicken"
+	force_string = "hilarious"
+	force = 10
+	damtype = STAMINA
+	hitsound = 'sound/weapons/genhit1.ogg'  //could use a funnier
+	attack_verb_continuous = list("slaps", "smacks", "gahonks")
+	attack_verb_simple = list("slap", "smack", "gahonk")
+
+/obj/item/rubber_chicken/afterattack(atom/target, mob/user, proximity_flag, click_parameters)
+	. = ..()
+	if(!isliving(target))
+		return
+	var/mob/living/slapped = target
+	slapped.apply_status_effect(STATUS_EFFECT_PRANKED)
+
+/obj/item/rubber_chicken/attack_secondary(mob/living/victim, mob/living/user, params)
+	if(HAS_TRAIT(user, TRAIT_PACIFISM))
+		to_chat(user, "<span class='warning'>You are a pacifist.</span>")
+		return SECONDARY_ATTACK_CANCEL_ATTACK_CHAIN
+
+	if(!HAS_TRAIT(user, TRAIT_CLUMSY))
+		return SECONDARY_ATTACK_CALL_NORMAL
+
+	var/datum/status_effect/pranked/victim_pranked = victim.has_status_effect(STATUS_EFFECT_PRANKED)
+	if(!victim_pranked)
+		return SECONDARY_ATTACK_CALL_NORMAL
+
+	if(victim.has_status_effect(STATUS_EFFECT_SLAPPED_SILLY))
+		return SECONDARY_ATTACK_CALL_NORMAL
+
+	var/slap_power = victim_pranked.prank_counter
+
+	playsound(user, 'sound/weapons/punch1.ogg', min(slap_power * 10, 80))
+	user.do_attack_animation(src)
+	victim.apply_damage(7 * slap_power, STAMINA, BODY_ZONE_HEAD)
+	victim.apply_status_effect(STATUS_EFFECT_SLAPPED_SILLY, slap_power SECONDS)
```

Comment:
slap_power SECONDS is unacceptable, make this a var beforehand named "slapped_silly_duration" or something like that and plug that in.

also, isn't prank counter... a counter? you're treating the counter as deciseconds? i don't like it but i don't think anyone will stop you